make-release.sh

```diff
@@ -136,6 +136,14 @@ if [[ $TRIGGER_RELEASE -eq 1 ]]; then
   git branch release -f 
   git push origin release -f
 
+  # build and publish devfile registry image
+  SHORT_SHA1="$(git rev-parse --short HEAD)"
+  docker build -t che-plugin-registry -f ./build/dockerfiles/Dockerfile --target registry .
+  docker tag che-plugin-registry quay.io/eclipse/che-plugin-registry:"${SHORT_SHA1}"
```

Comment:
why not skip a step here and just use 

```
docker build -t quay.io/eclipse/che-plugin-registry:"${SHORT_SHA1}" -f ./build/dockerfiles/Dockerfile --target registry .
docker push quay.io/eclipse/che-plugin-registry:"${SHORT_SHA1}"

```

then instead of tagging and pushing again:

```
  skopeo copy --all docker://quay.io/eclipse/che-plugin-registry:"${SHORT_SHA1}" docker://quay.io/eclipse/che-plugin-registry:"${VERSION}" 
```

Reply:
Also could we use build-in gh action for this, such that we can benefit from dockerx multiarch builds?

Reply:
as discussed elsewhere https://github.com/eclipse/che-devfile-registry/pull/309#pullrequestreview-546513013 moving to use dockerx is a future problem for another time.